SdsRestApiCore/SdsInterpolationMode.cs

```diff
@@ -2,7 +2,6 @@
 {
     public enum SdsInterpolationMode
     {
-        Default = Continuous,
```

Comment:
Why is this being removed?

Reply:
Style guide warning, 0 is the default by definition so it is unnecessary to state it explicitly